packages/paste-website/src/components/tokens-list/NoTokensFound.tsx

```diff
@@ -5,27 +5,25 @@ import {Heading} from '@twilio-paste/heading';
 import {Card} from '@twilio-paste/card';
 import {Button} from '@twilio-paste/button';
 import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
 
 interface NoTokensFoundProps {
   onClearSearch: () => void;
 }
 
 export const NoTokensFound: React.FC<NoTokensFoundProps> = ({onClearSearch}) => (
   <Card data-cy="tokens-empty-state" padding="space150">
-    <Stack orientation="horizontal" spacing="space110">
-      <Box
-        as="img"
-        src="/images/patterns/empty-no-results-found.png"
-        alt="No results found illustration"
-        size="size20"
-      />
+    <Box display="flex" columnGap="space110">
+      <Box as="img" src="/images/no-tokens-found.png" alt="No results found illustration" size="size20" />
```

Comment:
Nit: this image is decorative because the text below says the same thing. You can do alt="" instead.